templates/default/indicators.html

```diff
@@ -1,6 +1,31 @@
-{block name=indicatorcontents}
-<a href="?m=eventlist"><img src="img/calendar.gif" alt="{$eventcount.today}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/24-ring-red.png" alt="{$ticketcount.critical}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/ring-orange.png" alt="{$ticketcount.high}"></a>&nbsp;
-<a href="?m=rtqueueview"><img src="img/ring-blue.png" alt="{$ticketcount.unread}"></a>&nbsp;
+{block name=indicators}
```

Comment:
no i tu powinno być indicatorcontents, zamiast indicators, bo indicators to blok nadrzędny z zewnętrznym elementem &lt;div&gt;

Reply:
Ok poprawie jutro.